client/js/lounge.js

```diff
@@ -205,6 +207,7 @@ $(function() {
 			"action",
 		].indexOf(type) !== -1) {
 			switch (type) {
+			case "invite": data.msg.formattedAction = "invited " + data.msg.target + " to"; break;
```

Comment:
I don't usually like this sort of future proofing because it adds crust and confusion, but since it's already there and I believe you will add IRCv3 stuff as soon as you can, I'll pass on this one... :-)